[2024-11-02T07:38:08.940+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_load_to_postgres_reza.spark_submit_task manual__2024-11-02T07:38:08.355279+00:00 [queued]>
[2024-11-02T07:38:08.950+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_load_to_postgres_reza.spark_submit_task manual__2024-11-02T07:38:08.355279+00:00 [queued]>
[2024-11-02T07:38:08.951+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-11-02T07:38:08.967+0000] {taskinstance.py:2214} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2024-11-02 07:38:08.355279+00:00
[2024-11-02T07:38:08.971+0000] {standard_task_runner.py:60} INFO - Started process 6934 to run task
[2024-11-02T07:38:08.974+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'spark_load_to_postgres_reza', 'spark_submit_task', 'manual__2024-11-02T07:38:08.355279+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/spark-loadpostgres-reza.py', '--cfg-path', '/tmp/tmpf1mziqw4']
[2024-11-02T07:38:08.977+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask spark_submit_task
[2024-11-02T07:38:08.995+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.9/site-packages/airflow/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-02T07:38:09.031+0000] {task_command.py:423} INFO - Running <TaskInstance: spark_load_to_postgres_reza.spark_submit_task manual__2024-11-02T07:38:08.355279+00:00 [running]> on host dataeng-airflow-scheduler
[2024-11-02T07:38:09.104+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Reza' AIRFLOW_CTX_DAG_ID='spark_load_to_postgres_reza' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T07:38:08.355279+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-02T07:38:08.355279+00:00'
[2024-11-02T07:38:09.110+0000] {base.py:83} INFO - Using connection ID 'spark_main' for task execution.
[2024-11-02T07:38:09.112+0000] {spark_submit.py:473} INFO - Spark-Submit cmd: spark-submit --master spark://dataeng-spark-master:7077 --jars /spark-scripts/jars/jars_postgresql-42.2.20.jar --name arrow-spark /spark-scripts/spark-loadto-postgres-reza.py
[2024-11-02T07:38:11.470+0000] {spark_submit.py:634} INFO - 24/11/02 07:38:11 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-11-02T07:38:11.758+0000] {spark_submit.py:634} INFO - python3: can't open file '/spark-scripts/spark-loadto-postgres-reza.py': [Errno 2] No such file or directory
[2024-11-02T07:38:11.766+0000] {spark_submit.py:634} INFO - 24/11/02 07:38:11 INFO ShutdownHookManager: Shutdown hook called
[2024-11-02T07:38:11.766+0000] {spark_submit.py:634} INFO - 24/11/02 07:38:11 INFO ShutdownHookManager: Deleting directory /tmp/spark-41d42b48-604e-4144-9009-e5979cd3fa03
[2024-11-02T07:38:11.805+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 176, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 560, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://dataeng-spark-master:7077 --jars /spark-scripts/jars/jars_postgresql-42.2.20.jar --name arrow-spark /spark-scripts/spark-loadto-postgres-reza.py. Error code is: 2.
[2024-11-02T07:38:11.809+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=spark_load_to_postgres_reza, task_id=spark_submit_task, execution_date=20241102T073808, start_date=20241102T073808, end_date=20241102T073811
[2024-11-02T07:38:11.821+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 14 for task spark_submit_task (Cannot execute: spark-submit --master spark://dataeng-spark-master:7077 --jars /spark-scripts/jars/jars_postgresql-42.2.20.jar --name arrow-spark /spark-scripts/spark-loadto-postgres-reza.py. Error code is: 2.; 6934)
[2024-11-02T07:38:11.836+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-11-02T07:38:11.849+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
